scanner/cmd/scanner/main.go

```diff
@@ -72,30 +71,14 @@ func main() {
 
 	// Wait for signals.
 	sigC := make(chan os.Signal, 1)
-	signal.Notify(sigC, syscall.SIGINT, syscall.SIGTERM)
+	signal.Notify(sigC, os.Interrupt, unix.SIGTERM)
```

Comment:
Care to clarify why?

Reply:
copying over from https://github.com/stackrox/stackrox/pull/6663#discussion_r1266040876:

> something I forgot: I don't think syscall is really supposed to be used directly anymore: https://pkg.go.dev/syscall#pkg-overview
>
> The current Scanner v2 and Sensor both watch os.Interrupt, unix.SIGTERM - stackrox/rox#5024

Reply:
I see, let's do this then:

```
signal.Notify(sigC, unix.SIGINT, unix.SIGTERM)
```